lib/galaxy/objectstore/s3_multipart_upload.py

```diff
@@ -14,17 +14,8 @@
 
 from multiprocessing.pool import IMapIterator
 
-try:
-    from galaxy import eggs
-    eggs.require('boto')
-except ImportError:
-    pass
-
-try:
-    import boto
-    from boto.s3.connection import S3Connection
-except ImportError:
-    boto = None
+import boto
+from boto.s3.connection import S3Connection
```

Comment:
@natefoo - Yeah, even if we made a boto a required dependency there is no harm in letting something like the object store module be usable without it. By the same token yaml is no longer an optional dependency in pulsar, but I still like the ``ImportError`` and throw a more specific message at startup time thing in case we ever want to reuse portions of the code in different places.